docker/secondary-analysis-python/tools/submit/submit/confirm_submission.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+import requests
+import json
+import argparse
+import time
+
+def run(envelope_url, retry_seconds, timeout_seconds):
+    start = time.time()
+    current = start
+    while True:
+        print('Getting status for {}'.format(envelope_url))
+        envelope_js = get_envelope_json(envelope_url)
+        status = envelope_js.get('submissionState')
+        print('submissionState: {}'.format(status))
+        if status == 'Valid':
+            break
+        time.sleep(retry_seconds)
+        current = time.time()
+        if current - start >= timeout_seconds:
+            message = 'Timed out while waiting for Valid status. Timeout seconds: {}'.format(timeout_seconds)
+            raise ValueError(message)
+    confirm(envelope_url)
+
+def confirm(envelope_url):
+    print('Confirming submission')
+    headers = {
+        'Content-type': 'application/json'
+    }
+    response = requests.put('{}/submissionEvent'.format(envelope_url), headers = headers)
+    print(response.text)
+
+def get_envelope_json(envelope_url):
+    response = requests.get(envelope_url)
+    envelope_js = response.json()
+    return envelope_js
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-envelope_url', required=True)
+    parser.add_argument('-retry_seconds', type=int, required=True)
+    parser.add_argument('-timeout_seconds', type=int, required=True)
```

Comment:
For retry_seconds and timeout_second. is it possible to give default values?  just curious

Reply:
Good idea, will do!